cove/cove_360/templatetags/cove_tags.py

```diff
@@ -0,0 +1,16 @@
+from cove.html_error_msg import html_error_msg
+from cove.templatetags.cove_tags import register
+from django.utils.translation import gettext_lazy as _
+
+
+@register.filter(name='html_error_msg')
```

Comment:
Just wondering why this filter is being added? It appears to duplicate what is happening in the validation_table.html template, i.e.:

```jinja
{% if error.error_id ==  "uniqueItems_with_id" %}
        <a href="https://standard.threesixtygiving.org/en/latest/identifiers/#grant-identifier" target="_blank">(more info)</a>
{% endif %}
```

So it could just be written as
```jinja
 {% if error.error_id ==  "uniqueItems_with_id" %}
<p>{% blocktrans %}Non-unique id values. There are grants in your data with the same Id. [ ... ] {% endblocktrans %}</p>

        <a href="https://standard.threesixtygiving.org/en/latest/identifiers/#grant-identifier" target="_blank">(more info)</a>

{% endif %} 
```
?

Reply:
@michaelwood  Yes, it could just be written like that. I prefer it in the way I have done it as the filter replaces the default message and I find it more clear having it in a separate `cove_tags` file. If there is a nicer way to add the `more info` link in the `cove_tags` file, then I can move it to there.

Reply:
@michaelwood I have moved everything to `cove_tags`